src/circuit_builder.rs

```diff
@@ -1258,15 +1261,25 @@ impl<C: HaloCurve> CircuitBuilder<C> {
         let constants_coeffs: Vec<Vec<C::ScalarField>> =
             values_to_coeffs(&wire_constants, &fft_precomputation_n);
         let constants_8n = coeffs_to_values_padded(&constants_coeffs, &fft_precomputation_8n);
-        let c_constants = coeffs_to_commitments(&constants_coeffs, &pedersen_g_msm_precomputation);
+        let c_constants = PolynomialCommitment::coeffs_vec_to_commitments(
+            &constants_coeffs,
+            &pedersen_g_msm_precomputation,
+            pedersen_h,
+            true,
```

Comment:
This can be false right (and below for c_s_sigmas)? It probably doesn't matter for performance, but it might be best if our circuit-building code is deterministic.

Reply:
I think always-false seems good -- I figure if someone wants to mask what circuit they're using, it'd probably be better to commit to the VK (like in Zexe) rather than using a blinded VK, so that each commit can use different randomness.